swap_meet/vendor.py

```diff
@@ -1,2 +1,67 @@
+from .item import Item
+
 class Vendor:
-    pass
+    def __init__(self, inventory = None):
+        if inventory:
+            self.inventory = inventory
+        else:
+            self.inventory = []
+    
+    def add(self, item):
+        self.inventory.append(item)
+        return item
+    
+    def remove(self, item):
+        if item in self.inventory:
+            self.inventory.remove(item)
+            return item
+        else:
+            return False
+    
+    def get_by_category(self, category):
+        category_items = []
+        for item in self.inventory:
+            if item.category == category:
+                category_items.append(item)
+        
+        return category_items
```

Comment:
Great work!

We can refactor this for loop into a list comprehension:

```suggestion
    def get_by_category(self, category):
        category_items = [ item for item in self.inventory if item.category == category]

        return category_items
```